src/backend/dx12/src/window.rs

```diff
@@ -98,10 +103,13 @@ pub struct Swapchain {
     pub(crate) frame_queue: VecDeque<usize>,
     #[allow(dead_code)]
     pub(crate) rtv_heap: n::DescriptorHeap,
+    // need to associate raw image pointers with the swapchain so they can be properly released
+    // when the swapchain is destroyed
+    pub(crate) resources: Vec<*mut d3d12::ID3D12Resource>,
```

Comment:
shouldn't this be using `ComPtr` for releasing?

Reply:
I inherited this from hriundel's changes. The correct usage is beyond me, you should ask someone with d3d12 experience.

Reply:
yes should be a ComPtr